app/jobs/scheduled/turn_off_do_not_disturb.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jobs
+  class TurnOffDoNotDisturb < ::Jobs::Scheduled
+    every 1.minute
+
+    def execute(args = nil)
+      now = Time.current
+      DoNotDisturbTiming.includes(:user).where('ends_at <= ? AND ends_at > ?', now, now - 1.minute).each do |timing|
```

Comment:
This is assuming the job gets executed within the minute. What if jobs are paused or behind schedule?

It would be better to add another field to the model to indicate if you've sent a notification yet (either a `notified` bool or `notified_at` timestamp). Then you could query all timings that are over but not notified (make sure you index both fields.)